tests/test_lib_caconnector.py

```diff
@@ -468,10 +469,10 @@ def test_01_create_ca_connector(self):
             self.assertEqual(description[key], "string")
 
         # Check, if an error is raised if a required attribute is missing:
-        self.assertRaisesRegexp(CAError, "required argument 'port' is missing.",
-                                MSCAConnector, "billsCA", {MS_ATTR.HOSTNAME: "hans"})
-        self.assertRaisesRegexp(CAError, "required argument 'hostname' is missing.",
-                                MSCAConnector, "billsCA", {MS_ATTR.PORT: "shanghai"})
+        self.assertRaisesRegex(CAError, "required argument 'port' is missing.",
```

Comment:
Please stick with `assertRaisesRegexp()` for now. We can change it when we officially drop Python 2.

Reply:
Oh, I changed this at a lot of places, I think.
Didn't we already drop the test cases for Python2?

Reply:
Nope, just for Python 3.5